cdap-master/src/main/java/co/cask/cdap/master/environment/k8s/StorageMain.java

```diff
@@ -73,6 +76,7 @@ void createStorage(CConfiguration cConf) throws IOException {
         protected void configure() {
           bind(AuthorizationEnforcer.class).to(NoOpAuthorizer.class);
           bind(TransactionSystemClient.class).to(ConstantTransactionSystemClient.class);
+          bind(MetricsCollectionService.class).to(NoOpMetricsCollectionService.class).in(Scopes.SINGLETON);
```

Comment:
This is not part of the test code. Shouldn't the real metrics collection service be bound here?

Reply:
I believe at this point the metrics collection service might not get started, I am not sure if it is right to put the actual metrics collection service. We only use this to create the system tables, so no table operation and transaction will get started

Reply:
This class is only used for storage creation, so probably tracking metrics is not very important.

Reply:
In that case, just add comment explaining why it is no-op metrics here